app/components/Poll/index.js

```diff
@@ -64,17 +64,39 @@ class Poll extends React.Component<Props, State> {
       : this.setState({ optionsToShow: options, expanded: true });
   };
 
+  optionsWithRatios = (options, optionsToShow) => {
+    const totalVotes = options.reduce((a, option) => a + option.votes, 0);
```

Comment:
This could be an argument to the function as we do have this value precomputed. However, I believe this results in cleaner code. I would have loved to remove one of the arguments but both is needed to replicate the existing code.

Why would you only show some options but still have their ratios calculated based on the total number of votes? It would make the poll look even weirder.